src/remarx/quotation/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+This module contains functionality for embedding generation and quotation detection.
```

Comment:
Might as well use similar language as the`sentence` module
```suggestion
This module contains libraries for embedding generation and quotation detection.
```